src/environments/environment.ts

```diff
@@ -13,7 +13,8 @@ export const environment = {
     "AIzaSyDHAAKMjGUxxQbZCeHJDCpAeLmQ2A7HqOI",
     ['https://slides.googleapis.com/$discovery/rest?version=v1'],
     'https://www.googleapis.com/auth/presentations'
-  ]
+  ],
+  SENTRY_ENV: 'test'
```

Comment:
Does this make it so errors encountered locally are logged to sentry TEST?

Reply:
This file would be replaced with environment.prod.ts: https://github.com/ithaka/aiw-ui/blob/7a7e916fafeab081dad390b14f3ad8d018d59b48/angular.json#L59-L65

Reply:
I would be down for a "dev" option too, just to get a little more granularity when debugging.

Reply:
Right, I think the prod one looks correct, but I am guessing this one is for localhost? Ya I think `dev` will work. But do we still get a visible error in the console when we log to sentry though? My fear is that sentry will swallow our errors locally instead of being aware of them.